cms/djangoapps/api/v1/tests/test_serializers/test_course_runs.py

```diff
@@ -24,14 +24,10 @@ class CourseRunSerializerTests(ModuleStoreTestCase):
     def test_data(self, expected_pacing_type, self_paced):
         start = datetime.datetime.now(pytz.UTC)
         end = start + datetime.timedelta(days=30)
-        enrollment_start = start - datetime.timedelta(days=7)
-        enrollment_end = end - datetime.timedelta(days=14)
 
         course = CourseFactory(
             start=start,
             end=end,
-            enrollment_start=enrollment_start,
```

Comment:
If we keep them as is with the `CourseFactory` I am sure the test would pass, but can you test it?

Reply:
sure @awaisdar001